content/docs/guides/nextjs.md

```diff
@@ -344,7 +344,7 @@ export const config = {
 
 ## Run the app
 
-For API handlers, and server functions you can expect to see one of the following in your terminal output:
```

Comment:
Not sure if we need to explicitly say this applies to both API Handlers and Server functions. Including that made this sentence feel a bit clunky. Watta you think @danieltprice / @bgrenon?

Reply:
I probably wouldn't include it. I like how you've worded it. "When you run..." is nice language, it assumes the reader knows the command already, but also gets it on paper for anyone who needs the reminder.